### PR TITLE
feat: create event repository for event service

### PR DESCRIPTION
**Pull request by ZixMai**
Create event repository
Edit docstrings
Edit import format
Add some typing
Change singleton class
Change prisma schema

### CODE REVIEW

File: event_service/repository/event_repository.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            Catch all for every exception raised by Prisma Client Python.
+        ValueNotFoundError
+            No events was found for given author id.
+
+        """
+        db_events: Optional[
+            List[PrismaEvent]
+        ] = await self._db_client.db.event.find_many(where={"id": author_id})
+        if db_events is None:
+            raise ValueNotFoundError("Events not found")
```
Comment (Nhsdkk) [inline event_service/repository/event_repository.py:67]: Check for empty array, and throw ValueNotFoundError if its empty

**Review by Nhsdkk — CHANGES_REQUESTED** [review]
Check review and make changes accordingly

**Review by Nhsdkk — APPROVED** [review]
LGTM

File: event_service/repository/event_repository.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        prisma.errors.PrismaError
+            Catch all for every exception raised by Prisma Client Python.
+        ValueNotFoundError
+            No events was found for given author id.
+
+        """
+        db_events: Optional[
+            List[PrismaEvent]
+        ] = await self._db_client.db.event.find_many(where={"id": author_id})
+        if db_events is None or len(db_events) == 0:
```
Comment (MaxPlays35) [inline event_service/repository/event_repository.py:66]: It's okay, according to documentation of python https://docs.python.org/3/reference/expressions.html#boolean-operations

**Review by MaxPlays35 — APPROVED** [review]
Approved

**Pull request merged**